rsconf/package_data/base_os/main.sh.jinja

```diff
@@ -114,16 +115,28 @@ base_os_ipv4() {
         sysctl -w net.ipv6.conf.all.disable_ipv6=1 net.ipv6.conf.default.disable_ipv6=1
         dracut -f
     fi
-    base_os_postfix
     base_os_pwquality
     base_os_rpcbind_patch
     return $no_reboot
 }
 
 base_os_postfix() {
-    if type postconf >& /dev/null; then
+    if ! type postconf >& /dev/null; then
```

Comment:
One concern with this: since we have to remove manually when re-configuring a host it seems possible a host could be configured for postfix, then configured for another purpose, and postfix rpm won't be uninstalled. Is there another test we can do?

Reply:
It's not a problem if the lines below get executed. If postfix is removed, then iptables will close the port.

Reply:
Actually the lines below need to be execute when there is no postfix component. Note the else.